internal/node/generate_build_file.js

```diff
@@ -56,12 +56,7 @@ function main() {
 module.exports = { main };
 
 function generateBuildFile() {
-  // *.pyc files are generated during node-gyp compilation and include
-  // absolute paths, making them non-hermetic.
-  // See https://github.com/bazelbuild/rules_nodejs/issues/347
-  const excludedFiles = ['.md', '.html', '.pyc'];
```

Comment:
where did this code go? Don't we still need to exclude these non-hermetic inputs?

Reply:
This is just bringing the changes that you did here: https://github.com/bazelbuild/rules_nodejs/pull/283/commits/3380dd712c187f36757f41ded4047ea91aa3bfe6#diff-67b0f7c2b38c3bbb6dd40c85eb923174L242 up-to-date with the new code. Basically `node_runfiles` includes a whole load of extra stuff like yarn and everything else that node ships with but imo that seems unnecessary, as if someone e.g. needs yarn they should just add it as a dependency explicitly rather than everyone having to pay the cost for this. And our own internal app is still working fine after this change and all tests here are still passing so it seems to me these files are not needed for a `nodejs_binary` and they add quite a bit of weight to the resulting docker image.